controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobProfile.java

```diff
@@ -64,6 +68,7 @@ public static JobProfile of(JobType type) {
             case test: return systemTest;
             case staging: return stagingTest;
             case prod: return production;
+            case dev: return development;
```

Comment:
`case perf`?

Reply:
There are no jobs for perf yet. Should we add those right away?

Reply:
Guess I can do that.